CVLByExample/Invariant/RequireInvariantArray/README.md

```diff
@@ -0,0 +1,60 @@
+# Sorted Array Invariant Example
+
+This folder illustrates how the **new** `requireInvariant` semantics handle a **sorted array** property more accurately than the old semantics. We have:
+
+## Key Points
+
+- **Old semantics**: Using `requireInvariant` in non-boundary contexts (e.g., hooks, summaries) could lead to false verification results. The invariant might be incorrectly considered satisfied inline, missing real violations.
+- **New semantics**: `requireInvariant` is enforced only at valid rule boundaries (e.g., after function calls, subcalls, or havocs if `strong`). This ensures that violations are properly detected.
```

Comment:
@nd-certora I'd like to double check what happens in the case of a `strong` invariant that is havoced, do you have a good example here? I attempted to create one for testing but wasn't yet succeeding. Do you have an idea?